iterator_massive.py

```diff
@@ -0,0 +1,34 @@
+class MassiveIterator:
+    def __init__(self, massive, start=0, step=1):
```

Comment:
What exactly is `massive`? Let's add docs.

Reply:
Now you can check the docs using "print(MassiveIterator.__doc__)". Thanks!
The same was done for MassiveGenerator.